indico/modules/categories/controllers/display.py

```diff
@@ -299,24 +299,25 @@ def _process(self):
         managers = sorted(self.category.get_manager_list(), key=attrgetter('principal_type.name', 'name'))
 
         threshold_format = '%Y-%m'
-        params = {'event_count': len(events),
-                  'events_by_month': events_by_month,
-                  'format_event_date': self.format_event_date,
-                  'future_event_count': future_event_count,
-                  'show_future_events': show_future_events,
-                  'future_threshold': future_threshold.strftime(threshold_format),
-                  'happening_now': self.happening_now,
-                  'is_recent': self.is_recent,
-                  'managers': managers,
-                  'past_event_count': past_event_count,
-                  'show_past_events': show_past_events,
-                  'past_threshold': past_threshold.strftime(threshold_format),
-                  'json_ld': map(serialize_event_for_json_ld, json_ld_events),
-                  'atom_feed_url': url_for('.export_atom', self.category),
-                  'atom_feed_title': _('Events of "{}"').format(self.category.title)}
+        params = {
```

Comment:
This is just a format change now so I'd fully omit it

Reply:
Fine, omitted.